crates/bevy_log/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "bevy_log"
+version = "0.3.0"
+edition = "2018"
+authors = [
+    "Bevy Contributors <bevyengine@gmail.com>",
+    "Carter Anderson <mcanders1@gmail.com>",
+]
+description = "Provides logging for Bevy Engine"
+homepage = "https://bevyengine.org"
+repository = "https://github.com/bevyengine/bevy"
+license = "MIT"
+keywords = ["bevy"]
+
+
+[dependencies]
+bevy_app = { path = "../bevy_app", version = "0.3.0" }
+bevy_utils = { path = "../bevy_utils", version = "0.3.0" }
+
+tracing-subscriber = "0.2.15"
+tracing-chrome = { git = "https://github.com/superdump/tracing-chrome", branch = "name.fields", optional = true}
```

Comment:
This still points to my branch. :)

Reply:
Haha oops. I threw that in for testing and forgot to revert it 😄